package.json

```diff
@@ -2,15 +2,15 @@
   "private": true,
   "name": "zenorocha.com",
   "version": "5.0.0",
+  "license": "MIT",
   "scripts": {
     "dev": "next dev",
     "build": "next build",
-    "lint": "next lint",
+    "lint": "echo \"\" && echo \"› eslint\" && eslint --fix . && echo \"› prettier\" && prettier --write --log-level warn . && echo \"✔ clean\" && echo \"\"",
```

Comment:
We don't need all these `echo`s, let's keep it simple with only the necessary commands